presto-main/src/main/java/com/facebook/presto/operator/scalar/VarbinaryFunctions.java

```diff
@@ -252,6 +252,14 @@ public static Slice md5(@SqlType(StandardTypes.VARBINARY) Slice slice)
         return computeHash(Hashing.md5(), slice);
     }
 
+    @Description("compute hash equivalent to MurmurHash3_x64_128 (Murmur3F) in C++")
+    @ScalarFunction("murmurhash3X64128")
```

Comment:
```suggestion
    @ScalarFunction("murmur3_x64_128")
```
This is the name that people will use to invoke it so we don't want this to be some weird camel case thing. We want it to be consistent with the names of the other functions.

I think we also do want to drop `hash` from `murmurhash3` because I think everyone knows what `murmur3`  is so adding the hash is redundant. It's also a little less typing for people when they are authoring queries.